src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php

```diff
@@ -37,6 +37,7 @@ protected function configure()
             ->setDescription('Lints a file and outputs encountered errors')
             ->addArgument('filename', null, 'A file or a directory or STDIN')
             ->addOption('format', null, InputOption::VALUE_REQUIRED, 'The output format', 'txt')
+            ->addOption('exclude_dir', null, InputOption::VALUE_OPTIONAL, 'Exclude a list of directory', '')
```

Comment:
`VALUE_OPTIONAL` doesn't make sense to me. When would you want to use this option without any value?